site/docs/transform/aggregate.md

```diff
@@ -14,7 +14,7 @@ To aggregate data in Vega-Lite, users can either use use an `aggregate` property
 
 
 {:#encoding}
-## Aggregate (Encoding)
+## Aggregate Encoding
```

Comment:
Let's call this "Aggregate in Encoding Field Definition"

(Please feel free to do the same correction for bin and timeUnit)

Reply:
Done for all.